packages/app-staking/src/StakeList/Nominating.tsx

```diff
@@ -104,17 +109,55 @@ class Nominating extends React.PureComponent<Props> {
           onChange={this.onChangeNominee}
           value={nominee}
         />
+        {
+          !isNomineeValid
+          ? <Labelled>
```

Comment:
Not entirely excited about the use of this since we basically ignore what it is for however, we don't have a box-y component, so you made do with what you had. No issues there.

(We probably need something in ui-app that does a "display a box with border", e.g. if you look at Transfer there is something that displays the fees, but completely local to there).

TL;DR - Not excited, but it is ok since there is little alternative atm

Reply:
We could add an optional prop to the Labelled component for the semantic-react-ui's Label props of color and pointing: https://react.semantic-ui.com/elements/label/#types-pointing-colored

Reply:
Ahh, yes, I actually pointed Luke to that a while back. We should obviously not over-use it, but I think it makes perfect sense.

Maybe not use Labelled for that since it is actually different from the SUI "Label", but rather call it something else and just wrap that properly. (Not sure what on the naming). In Labelled we actually fight against the SUI Label, the intent is different.